packages/puppeteer-core/src/common/Frame.ts

```diff
@@ -63,7 +63,6 @@ export class Frame extends BaseFrame {
   _frameManager: FrameManager;
   override _id: string;
   _loaderId = '';
-  override _hasStartedLoading = false;
```

Comment:
is it not used anymore?

Reply:
This leave on the api Frame class and as a `"protected"` value is shared with child classes.